instrumentation/kafka-streams/src/main/java/brave/kafka/streams/TracingProcessor.java

```diff
@@ -55,6 +56,7 @@ public void process(K k, V v) {
       delegateProcessor.process(k, v);
     } catch (RuntimeException | Error e) {
       error = e;
+      Call.propagateIfFatal(e);
```

Comment:
Nit might be better to use static import - seeing `Call` out of nowhere could trigger some confusion. Better would be if we moved the method to a separate utility at some point though :)